build.sbt

```diff
@@ -314,6 +314,15 @@ lazy val core = (project in file("core"))
   .dependsOn(macros)
   .dependsOn(firrtl)
 
+lazy val generateBoilerplate = taskKey[Unit]("generate boilerplate code for the core project")
```

Comment:
@jackkoenig is it alright if I make this a standalone sbt task that you run manually? since developers probably already have sbt installed, and that way we can write files directly into the scala directory without requiring cli args